logos/logos-workernode/tests/test_vllm_process.py

```
@@ -425,6 +425,53 @@ def test_build_cmd_omits_max_model_len_when_no_profile(monkeypatch) -> None:
+    hybrid Mamba/SSM models — without it the lane reverts to vLLM's default
+    1024 and aborts CUDA-graph capture at startup."""
+    from logos_worker_node.model_profiles import ModelProfileRecord, ModelProfileRegistry
+
+    registry = ModelProfileRegistry()
+    registry._profiles["RedHatAI/Qwen3-Coder-Next-NVFP4"] = ModelProfileRecord(
+        engine="vllm",
+        calibration_max_num_seqs=160,
+    )
+
+    handle = VllmProcessHandle("lane-test", 19000, OllamaConfig(), model_profiles=registry)
+    monkeypatch.setattr(handle, "_resolve_vllm_binary", lambda _configured: "/tmp/vllm")
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/vllm"

(S108)